api/src/test/java/io/opentelemetry/api/baggage/ScopedBaggageTest.java

```diff
@@ -31,15 +30,12 @@
       EntryMetadata.create("unlimited");
   private static final EntryMetadata METADATA_NO_PROPAGATION = EntryMetadata.create("noprop");
 
-  @BeforeEach
-  void setUp() {
-    Context.root().makeCurrent();
-  }
-
   @Test
   void emptyBaggage() {
-    Baggage defaultBaggage = BaggageUtils.getCurrentBaggage();
-    assertThat(defaultBaggage.getEntries()).isEmpty();
+    try (Scope s = Context.root().makeCurrent()) {
```

Comment:
Let's remove the scope - just to make sure to test what happens when makeCurrent hasn't ever been called